packages/period-selector-dialog/css/PeriodSelector.css

```diff
@@ -113,6 +112,12 @@ button.nav-button {
 .selector-area {
     position: relative;
     height: 460px;
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
```

Comment:
I'm not sure its necessary to include the browser prefixes. Apps including the component should configure their bundler to do the prefixing (see webpack.config.prod.js in data-visualizer)

Reply:
got it, thanks, will fix now